parsimonious/grammar.py

```diff
@@ -420,8 +420,8 @@ def _resolve_refs(self, rule_map, expr, done):
                 # of `expr.members` can refer back to `expr`, but it can't go
                 # any farther.
                 done.add(expr)
-                expr.members = [self._resolve_refs(rule_map, member, done)
-                                for member in expr.members]
+                expr.members = tuple([self._resolve_refs(rule_map, member, done)
```

Comment:
The cast to a tuple actually makes it consistent with the other spots that write to it, so this makes me happy. Good correction.